src/main/java/com/dev/cinema/controllers/OrderController.java

```diff
@@ -31,16 +32,20 @@ public OrderController(OrderService orderService, OrderMapper mapper,
     }
 
     @GetMapping
-    public List<OrderResponseDto> getUserHistory(@RequestParam Long userId) {
-        return orderService.getOrderHistory(userService.getById(userId))
+    public List<OrderResponseDto> getUserHistory(Authentication authentication) {
+        UserDetails userDetails = (UserDetails) authentication.getPrincipal();
+        String email = userDetails.getUsername();
```

Comment:
Code duplicate, it is possible to extract this.

Reply:
You're right but I think both variants are ok